README.md

```diff
@@ -20,40 +20,43 @@
 ## Third Party Dependencies
 
 - snappy
-- lz4
 - zlib
 - thrift 0.7+ [install instructions](https://thrift.apache.org/docs/install/)
 - googletest 1.7.0 (cannot be installed with package managers)
 - Google Benchmark (only required if building benchmarks)
 
 You can install these dependencies using a package manager or using the
-`thirdparty/` scripts in this repository. On Homebrew, you can run:
+`thirdparty/` scripts in this repository.
+
+Mac already has zlib; on Homebrew, you can run:
+
+```shell
+ brew install snappy thrift
+```
+
+For Linux, just run
 
 ```shell
- brew install snappy lz4 thrift zlib
+source setup_build_env.sh
 ```
 
-To build the thirdparty libraries in-tree, run:
+By default, it will create a build directory `build/`. You can override the
+build directory by setting the BUILD_DIR env variable to another location.
+
+To manually build the thirdparty libraries in-tree, run:
```

Comment:
Do you want to mention that we can choose specific dependencies to be installed ?
`./thirdparty/build_thirdparty.sh zlib`
options: arrow, zlib, gbenchmark, gtest, snappy, thrift